### DISCUSSION TITLE
ServerOption changes

### DISCUSSION THREAD

**Discussion by prawnsalad** [lxzan/gws#91] [Q&A] [answered]
gws works awesome and I really appreciate the performance digging you do, thanks!

I'm upgrading to the latest gws version and noticed these ServerOption changes:
~~~golang
		// CompressEnabled:  true,         // Enable compression
		// ReadAsyncEnabled: true,         // Parallel message processing
		PermessageDeflate: gws.PermessageDeflate{
			Enabled: true,
		},
		ParallelEnabled: true,         // Enable parallel processing
~~~

Would these changes be pretty much the same? Or will the default values in PermessageDeflate cause something different? I can't find any docs or release notes on these changes

**Comment by lxzan** [comment, marked as answer]
ReadAsyncEnabled renamed to ParallelEnabled; CompressEnabled replaced by PermessageDeflate.Enabled.

gws v1.8 supports the full rfc 7692 feature, so the compression-related configurations have been put into permessage-deflate, the old version of gws only holds no_context_takeover mode.

**Comment by lxzan** [comment]
default values have no change.
